src/lib/database/populate-from-live-api.ts

```diff
@@ -0,0 +1,47 @@
+import type { Routes } from "@seamapi/types/devicedb"
+import axios from "axios"
+import type { Simplify } from "type-fest"
+import type { TypedAxios } from "typed-axios-instance"
+
+import type { Database } from "./schema.ts"
+
+interface PopulateFromLiveApiOptions {
+  db: Database
+  vercel_protection_bypass_secret: string
+  endpoint: string
+  device_category: Routes["/v1/device_models/list"]["queryParams"]["main_category"]
+}
+
+export const populateFromLiveApi = async ({
```

Comment:
Instead of populate (introducing a new concept), can we just call this `seedFromLiveApi` and change the interface to match the other seed function, e.g.,
```suggestion
export const seedFromLiveApi = async (db: Database, client: Axios, { device_category = 'lock' } = {}: Params
```

Reply:
6cecbe0